Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/W8-IndividualNodePublication/03-MoreBasicFeatures.feature

```diff
@@ -155,27 +158,73 @@ Feature: Publishing individual nodes (basics)
     And I expect this node to have the following properties:
       | Key  | Value         |
       | text | "Modified t2" |
-    Then I expect node aggregate identifier "sir-nodeward-nodington-iii" to lead to node user-cs-identifier-remaining;sir-nodeward-nodington-iii;{}
+    Then I expect node aggregate identifier "sir-nodeward-nodington-iii" to lead to node user-cs-identifier;sir-nodeward-nodington-iii;{}
     And I expect this node to have the following properties:
       | Key   | Value            |
       | image | "Modified image" |
 
+    # assert that content stream is still open by writing to it:
+    And the command SetNodeProperties is executed with payload:
+      | Key                       | Value                        |
+      | workspaceName             | "user-test"                  |
+      | nodeAggregateId           | "sir-nodeward-nodington-iii" |
+      | originDimensionSpacePoint | {}                           |
+      | propertyValues            | {"image": "Bla bli blub"}    |
+
+  Scenario: Tag the same node in live and in the user workspace so that a rebase will omit the user change
+    When the command TagSubtree is executed with payload:
+      | Key                          | Value                     |
+      | workspaceName                | "live"                    |
+      | nodeAggregateId              | "sir-unchanged"           |
+      | nodeVariantSelectionStrategy | "allVariants"             |
+      | tag                          | "tag1"                    |
+    When the command TagSubtree is executed with payload:
+      | Key                          | Value                     |
+      | workspaceName                | "user-test"               |
+      | nodeAggregateId              | "sir-unchanged"           |
+      | nodeVariantSelectionStrategy | "allVariants"             |
+      | tag                          | "tag1"                    |
+    When the command PublishIndividualNodesFromWorkspace is executed with payload:
+      | Key                             | Value                                                             |
+      | workspaceName                   | "user-test"                                                       |
+      | nodesToPublish                  | [{"dimensionSpacePoint": {}, "nodeAggregateId": "sir-unchanged"}] |
+      | contentStreamIdForRemainingPart | "user-cs-identifier-remaining"                                    |
+
+    When I am in workspace "live" and dimension space point {}
+    Then I expect node aggregate identifier "sir-unchanged" to lead to node cs-identifier;sir-unchanged;{}
+    And I expect this node to be exactly explicitly tagged "tag1"
+
+    When I am in workspace "user-test" and dimension space point {}
+    Then I expect node aggregate identifier "sir-unchanged" to lead to node user-cs-identifier-remaining;sir-unchanged;{}
```

Comment:
@kitsunet i removed our special handling again when the events that were generated are empty as this would make the code way to complex for no gain except that theoretical case ... and here due to the live change the workspace was outdated and needs a rebase either way ... so impossible ^^
The expected content stream is now `user-cs-identifier-remaining` and not unchanged!

see https://github.com/neos/neos-development-collection/pull/5301/commits/49d58709e9a40c90ff7abb69132f3cf972f4a103

Reply:
Fine by me :)